frontend/src/hooks/usePets.ts

```diff
@@ -1,9 +1,17 @@
 import {Pet} from "../model/Pet";
-import {useState} from "react";
+import {useEffect, useState} from "react";
 import axios from "axios";
 
 export default function usePets(){
     const [pets,setPets]=useState<Pet[]>([]);
+    function loadAllPets(){
+        axios.get("/api/pets/")
+            .then(response=>response.data
+                .map((pet:{})=>({...pet}))
+            )
```

Comment:
This mapping is not necessary. You could directly do:
`.then(response => setPets(response.data))`

Reply:
Thanks a lot for your suggestions :-)